ui/components/multichain/app-header/__snapshots__/app-header.test.js.snap

```diff
@@ -609,19 +609,7 @@ exports[`App Header unlocked state matches snapshot: unlocked 1`] = `
             <div
               class="mm-box"
               style="position: relative;"
-            >
```

Comment:
We should remove the data that caused this from https://github.com/MetaMask/metamask-extension/blob/6b53e422c675262282555bbbe5b9ae5dbc65a2b8/test/data/mock-state.json#L635-L650